app/src/main/java/org/oppia/app/home/HomeActivityPresenter.kt

```diff
@@ -1,15 +1,37 @@
 package org.oppia.app.home
 
+import android.view.View
 import androidx.appcompat.app.AppCompatActivity
+import androidx.appcompat.widget.Toolbar
+import androidx.drawerlayout.widget.DrawerLayout
 import org.oppia.app.R
 import org.oppia.app.activity.ActivityScope
+import org.oppia.app.drawer.NavigationDrawerFragment
 import javax.inject.Inject
 
 /** The presenter for [HomeActivity]. */
 @ActivityScope
 class HomeActivityPresenter @Inject constructor(private val activity: AppCompatActivity) {
+  var toolbar: Toolbar? = null
+  var navigationDrawerFragment: NavigationDrawerFragment? = null
+
+  fun init(title: String) {
+    toolbar = activity.findViewById<View>(R.id.toolbar) as Toolbar?
```

Comment:
Nit: I believe you want to use ``as? Toolbar`` not ``as Toolbar?``.

Reply:
Updated to Toolbar